ros_buildfarm/templates/devel/devel_job.xml.em

```diff
@@ -361,13 +367,21 @@ if pull_request:
 @(SNIPPET(
     'publisher_groovy-postbuild_maintainer-notification',
 ))@
-@[ end if]@
+@[end if]@
```

Comment:
I think this space was intentional to show the nesting level of the conditional without producing an artifact in the output file.

Reply:
Thanks. Fixed in dbbb07c